src/App.tsx

```diff
@@ -1,39 +1,67 @@
-import React from 'react';
+/* eslint-disable no-console */
+import { Component } from 'react';
 import './App.scss';
+import { Clock } from './components/Clock/Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+type State = {
+  hasClock: boolean,
+  clockName: string,
+  clockRandomNameId: number;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    hasClock: true,
+    clockName: 'Clock-0',
+    clockRandomNameId: 0,
```

Comment:
let's not store this value in state
just add it as separate variable, as you added state for the App class